docs/general/ecosystem/guides/guidelines.md

```diff
@@ -0,0 +1,166 @@
+- **Rationale**: Explain the reasoning behind your approach. It should cover alternative approaches considered, related work, and trade-offs made.
+- **Implementation:** For all proposals going through the governance process, this section should reference the code implementing the proposed change. It’s recommended to get community feedback before writing any code.
+- **Risks:** Highlight any risks and concerns that may affect consensus, proof-of-stake, governance, protocol economics, the stability protocol, security, and privacy.
+
+## 📄 Docs
+
+Technical writers support the Celo community by educating developers about Celo through engaging, informative, and insightful documentation.
+
+#### Edit an existing page
+
+To edit an existing page in the [documentation](https://docs.celo.org/), create a fork of the repo, commit your edits, and submit a PR.
+
+- Go to the page in the docs
+- Click **Edit this page** at the bottom of the page
+- Edit the page directly on GitHub
+- Describe the edit in the commit
+- Select “Create a new branch and start a pull request”
+- Describe changes in the Pull Request (PR)
+- Select "therealharpaljadeja", "viral-sangani" or "GigaHierz" as a reviewer
+- Changes must be approved and pass all of the site build checks before being merged.
+
+#### Add/remove pages
+
+To add a new page to the documentation, create a fork, add the new pages, and update the table of contents file to include your new pages in the appropriate location and submit a PR.
+
+- Add or delete pages directly in GitHub
+- Put new pages where you think makes the most sense, we can move them later
+- Create a PR to have your changes added to the live version of the site
+- Update the file called “sidebars.js” in the main folder
+- This file contains the site layout that you see on the left side of the docs site
+- Add or remove the appropriate files from the list
+
+## 🎓 Tutorials
+
+Write about your experience as a member of the Celo community, whether you're a CELO owner, developer, or project founder. Your experience and perspective are valuable and can help others.
+
+#### File naming
+
+Creating a new post in the blog is straightforward. Create a new file in the [blog directory](https://github.com/celo-org/docs/tree/main/blog) in the documentation repository. Filenames follow the format of `YYYY-MM-DD-post-name.md`. For example, this post was written November 8th, 2021 so it has the filename `2021-11-08-contributing.md`.
+
+#### Front Matter
+
+Posts are written in [Markdown](https://www.markdownguide.org/). Posts include front matter. The front matter is file metadata at the top of the file that provides more information about the post. The front matter for this post looks like this:
+
+```md
+---
+title: Contributing to the Blog
+description: How to contribute to the blog
+slug: blog-contributions
+authors:
+  - name: Josh Crites
+    title: Developer Relations, cLabs
+    url: https://github.com/critesjosh
```

Comment:
not sure it matters but do we want josh name to still be here?

Reply:
I've updated this with Octocat placeholder.